pyneuroml/plot/PlotSpikes.py

```diff
@@ -108,7 +108,16 @@ def process_args() -> argparse.Namespace:
     return parser.parse_args()
 
 
-def read_sonata_spikes_hdf5_file(file_name):
+def read_sonata_spikes_hdf5_file(file_name: str) -> dict:
+    """
+        Read spike times from a SONATA format HDF5 file.
+
+    Args:
+        file_name (str): The name of the HDF5 file.
```

Comment:
ah, good, but please follow the same format as the other docstrings

Reply:
Okay sir